scripts/init_db.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -ex # hanldling errors and Debuging purpose 
+# Declare environment variables
+
+DB_USER="${POSTGRES_USER:=postgres}"
+DB_PASSWORD="${POSTGRES_PASSWORD:=password}"
+DB_NAME="${POSTGRES_DB:=newsletter}"
+DB_PORT="${POSTGRES_PORT:=5432}"
+DB_HOST="${POSTGRES_HOST:=localhost}"
+SKIP_DOCKER=${SKIP_DOCKER:=false}
+DATABASE_URL="postgres://$DB_USER:$DB_PASSWORD@localhost:5432/$DB_DATABASE"
```

Comment:
What is the used of the defined variables above if you are still hardcoding values?
```suggestion
DATABASE_URL="postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}"
```
make sure that the script works